<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompDialog</class>
 <widget class="QDialog" name="CompDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Component Definition</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Prefix">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Type">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Value">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Import">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Prefix">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Prefix</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Value">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Import">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Build">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Add To Part Templates</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_units_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>x100 mil</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_PinLabels">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>135</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Labels</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_PinShape">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>135</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>135</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pin Type</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Height">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>195</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_Height">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>195</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Width">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>195</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Width">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>195</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Footprint Dimensions (x100 mil)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>225</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Footprint Point Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_PinNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Pin Number</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_PinNumber">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRGB">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>315</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose fill color</string>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Shape Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_units_9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>530</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>degrees</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_units_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>x100 mil</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_CY">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Centre Y</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Fill">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Fill shape with color</string>
   </property>
   <property name="text">
    <string>Fill</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CX">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Centre X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Line">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Draw shape outline (black)</string>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_CY">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_A1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_A3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DX">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_A2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Angle B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_A2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>500</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_DX">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Shape">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>315</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_A1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Angle A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Shape">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>315</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Shape</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_units_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>x100 mil</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_DY">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_units_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>x100 mil</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_DY">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>440</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_units_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>500</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>degrees</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_units_7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>470</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>degrees</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_CX">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="minimum">
    <double>-100.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_A3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>530</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonU">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>25</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Move shape towards front</string>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonD">
   <property name="geometry">
    <rect>
     <x>226</x>
     <y>280</y>
     <width>25</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Move shape towards back</string>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
